README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# HOMEWORK ONE   
+## Who is Paulino?
```

Comment:
Best practice is to leave blank lines between headers and content. It will avoid rendering issues when we build more complex R Markdown documents